coderd/testdata/insights/user-activity/multiple_users_and_workspaces_week_all_templates.json.golden

```diff
@@ -0,0 +1,41 @@
+{
+  "report": {
+    "start_time": "2023-08-15T00:00:00Z",
+    "end_time": "2023-08-22T00:00:00Z",
+    "template_ids": [
+      "00000000-0000-0000-0000-000000000001",
+      "00000000-0000-0000-0000-000000000002",
+      "00000000-0000-0000-0000-000000000003"
+    ],
+    "users": [
+      {
+        "template_ids": [
+          "00000000-0000-0000-0000-000000000001",
+          "00000000-0000-0000-0000-000000000002"
+        ],
+        "user_id": "00000000-0000-0000-0000-000000000004",
+        "username": "user1",
+        "avatar_url": "",
+        "seconds": 30540
```

Comment:
Note-to-self: Also verified this number, 1h is lost from `users[0].workspaces[1]` due to timeframe and all the minute intervals are summed from `users[0].workspaces[0]` giving us 8h + 29m. The hours in u0w0 overlap with u0w1 so we don't see an increase from those.